### PR TITLE
[msal-node]: Increase code coverage

### PR DESCRIPTION
**Pull request by samuelkubai**
Increase code coverage of the msal-node library to +90%

### CODE REVIEW

File: lib/msal-node/test/client/ConfidentialClientApplication.spec.ts [hunk 1]
```diff
@@ -133,4 +134,26 @@ describe('ConfidentialClientApplication', () => {
             expect.objectContaining(expectedConfig)
         );
     });
+
+    test.only('acquireTokenByClientCredential handles AuthErrors as expected', async () => {
```
Comment (jasonnutter) [inline lib/msal-node/test/client/ConfidentialClientApplication.spec.ts:138]:
```suggestion
    test('acquireTokenByClientCredential handles AuthErrors as expected', async () => {
```

Comment (jasonnutter) [reply lib/msal-node/test/client/ConfidentialClientApplication.spec.ts:138]: @samuelkubai FYI

**Review by jasonnutter — CHANGES_REQUESTED** [review]
Need to remove exclusive test.

**Comment by sameerag** [comment]
@samuelkubai any update on this?

**Comment by samuelkubai** [comment]
> @samuelkubai any update on this?

Hi @sameerag, working on this, this week whilst on call.

**Review by jasonnutter — APPROVED** [review]

**Review by hectormmg — APPROVED** [review]

**Pull request merged**